src/components/DatasetSearchListItem/README.md

```diff
@@ -0,0 +1,132 @@
+# DatasetSearchListItem Component
```

Comment:
Was this file generated manually? I don't disagree that we need a documentation system for our components - we've been considering storybook for awhile. I'm not sure how I feel about this format though.

Reply:
sorry this file was auto-generated. Removed.

https://github.com/GetDKAN/cmsds-open-data-components/pull/293/commits/c35cd4c270e7e92ea3ffa8ac42161d578fed70c5